.travis.yml

```diff
@@ -13,6 +13,16 @@ php:
   - 5.6
   - hhvm
 
-install: composer install
+services: mongodb
+
+env:
+  - MONGO_VERSION=stable
+
+before_install: >
+    [ $TRAVIS_PHP_VERSION = "hhvm" ] || (yes '' | pecl -q install -f mongo-${MONGO_VERSION} && echo "extension=mongo.so" >> `php --ini | grep "Loaded Configuration" | sed -e "s|.*:\s*||"`)
```

Comment:
If you're only testing against the stable `mongo` extension, does it make sense to just rely on the version that's already available on Travis? You should be able to just add the INI line (per: [this documentation](http://docs.travis-ci.com/user/languages/php/#PHP-extensions)).